app/src/components/attachments/AttachmentsList.tsx

```diff
@@ -200,23 +202,26 @@ const AttachmentsList: React.FC<IAttachmentsListProps> = (props) => {
                         {row.fileName}
                       </Link>
                     </TableCell>
+                    {!props.surveyId && <TableCell>{row.fileType}</TableCell>}
                     <TableCell>{getFormattedDate(DATE_FORMAT.ShortDateFormatMonthFirst, row.lastModified)}</TableCell>
                     <TableCell>{getFormattedFileSize(row.size)}</TableCell>
-                    <TableCell>
-                      <Box my={-1}>
-                        <Button
-                          color="primary"
-                          variant="text"
-                          startIcon={
-                            <Icon path={row.securityToken ? mdiLockOutline : mdiLockOpenVariantOutline} size={1} />
-                          }
-                          aria-label="toggle attachment security status"
-                          data-testid="toggle-attachment-security-status"
-                          onClick={() => showToggleSecurityStatusAttachmentDialog(row)}>
-                          {row.securityToken ? 'Secured' : 'Unsecured'}
-                        </Button>
-                      </Box>
-                    </TableCell>
+                    {!props.surveyId && row.securityToken && (
```

Comment:
The check on `row.securityToken` is making it such that, if the file is unsecure and securityToken is null, then no button is rendered, and so the user has no way of re-securing the file.